opts/mount.go

```diff
@@ -92,6 +93,11 @@ func (m *MountOpt) Set(value string) error {
 			mount.Type = mounttypes.Type(strings.ToLower(value))
 		case "source", "src":
 			mount.Source = value
+			if strings.HasPrefix(value, "."+string(filepath.Separator)) || value == "." {
```

Comment:
I guess as a follow-up, we should consider;

- also allowing `../` (relative parent directories)
- wondering if we should always do a `filepath.Clean()`

Reply:
For the second point, `filepath.Abs` already calls `Clean`